include/ck/tensor_operation/operator_transform/transform_conv_bwd_data_to_gemm_v1.hpp

```diff
@@ -60,6 +60,10 @@ make_out_n_ho_wo_k_grid_desc(const index_t N,
             return make_naive_tensor_descriptor_packed(make_tuple(N, Ho, Wo, K));
         }
     }
+    else
+    {
+        throw std::runtime_error("wrong! unsupported layout");
```

Comment:
Please make the error message more user friendly. You can print the layout name (It has overloaded `<<` operator) Take a look ie. here: https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/develop/include/ck/tensor_operation/gpu/device/impl/device_grouped_gemm_xdl_splitk_cshuffle.hpp#L392-L395

Reply:
Done.